.lintstagedrc

```diff
@@ -1,4 +1,4 @@
 {
-  "!(*eslintrc).{t,j}s{,x}": ["prettier --write", "eslint --fix"],
+  "!(*eslintrc|*.stories).{t,j}s{,x}": ["prettier --write", "eslint -c packages/elements/.eslintrc.js --fix"],
```

Comment:
Can you explain these changes?

Reply:
I think it was meant as a workaround to the linting problem we had before. We had no lint-Files in our storybook package was lead to errors when trying to commit. This got fixed in https://github.com/inovex/elements/pull/410.